src/main/java/com/kosa/mapbegood/domain/mymap/favorite/controller/FavoriteController.java

```diff
@@ -40,17 +44,52 @@ public FavoriteController(FavoriteService favoriteService) {
  
  @PostMapping("/create/{thememapId}")  //o
  public ResponseEntity<String> createFavorite(
-         @PathVariable Long thememapId,
+         @PathVariable(required=false) Long thememapId,
+         Authentication authentication) {
+	 try {
+	        if (thememapId == null) {
+	            return new ResponseEntity<>("테마맵 ID가 없습니다.", HttpStatus.BAD_REQUEST);
+	        }
```

Comment:
```suggestion
         @PathVariable Long thememapId,
         Authentication authentication) {
	 try {
```
required false 빼버리면 자동으로 bad request 발생해서
굳이 null 값으로 체크할 필요도 없을 거 같은데요??